Classes/Views/Constants.swift

```diff
@@ -37,6 +37,8 @@ enum Constants {
         static let bullet = "\u{2022}"
         static let bulletHollow = "\u{25E6}"
         static let search = NSLocalizedString("Search", comment: "")
+        static let searchGitHub = "\(Constants.Strings.search) GitHub"
+        static let searchBookmarks = "\(Constants.Strings.search) \(Constants.Strings.bookmarks)"
```

Comment:
Can we make these separate instead of composed? This wont work once localized.

Reply:
What won't work once localized?

Reply:
- "GitHub" is a plane string
- Most languages aren't just combos of words, "Search Bookmarks"
  - e.g. French, this translates to "rechercher des signets"
  - While doing them one-by-one in french, search is "chercher" and bookmarks is "signets", so combined is "chercher signets"

Reply:
Makes sense. The `GitHub` being a plane string didn't really seem like an issue to me, as that doesn't change based on language. I'll change.